README.md

```diff
@@ -674,3 +690,19 @@ Services to securely store your Docker images.
 [vimagick]: https://github.com/vimagick
 [weave]: https://github.com/weaveworks/weave
 [wsargent]: https://github.com/wsargent
+
+
+[c]: https://cdn.rawgit.com/akullpp/23246ca832bda82bb505230bf3538e2a/raw/d9bcdb769bf025292f9c6bc1290f01f1fcd1f864/commercial.svg
+[ancient]: ./icons/blind.png         
+[$]: ./icons/dollar.png        
+[heart]: ./icons/heart-red.png     
+[support]: ./icons/lifebuoy.png      
+[upcoming]: ./icons/rocket.png        
+[web]: ./icons/web.png
+[beta]: ./icons/bug.png           
```

Comment:
I'm not sure if a bug might not be misleading. 'Beta' might be buggy but its more 'non production ready'. I suggest to replace it by a badge with text. Something like [that](http://psdfinder.co/ui-kits/attractive-red-web-ribbon-tag-psd-freebie)?